lighthouserc.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  ci: {
+    collect: {
+      url: ['http://localhost:3000/'],
+      startServerCommand: 'make dev',
+    },
+    upload: {
+      target: 'temporary-public-storage',
```

Comment:
Question: would we consider to store lighthouse reports in our gcs bucket ?
(for the storage prcing problem, we could set storage duration time to automatically delete old reports)

Reply:
To address this issue (to store lighthouse reports), I followed the [guide](https://github.com/GoogleChrome/lighthouse-ci/tree/main/docs/recipes/docker-server#deployment) to build a lighthouse-ci server.

Currently, it runs as a gke service `lhci-server`.

Before proceeding further, a related PR which sets the nginx config is [here](https://github.com/twreporter/nginx-configs/pull/52), please have a see, thanks.

Reply:
Updated ([7341d5f](https://github.com/twreporter/twreporter-react/pull/2227/commits/7341d5f876eed1dfdf9c3122526804c580b2f1dd)), please have a see, thanks @Aylie-Chou.

ref: https://github.com/GoogleChrome/lighthouse-ci/blob/main/docs/configuration.md

Reply:
Except for CircleCI environment variables, I also kept a copy of the tokens in 1Password.